source/_posts/2020-07-01-release-113.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: post
-title: "0.113: Beta release notes"
-description: "0.113: Beta release notes"
-date: 2020-07-14 00:00:00
+title: "0.113: Automations & Scripts, and even more performance!"
+description: "0.113: Great new features to our Automations & Scripts, and even more performance improvements!"
```

Comment:
Could we try to not just reword the title but give some examples?

Reply:
Maybe: Run scripts in parallel, switch statements for automations and microsecond precision on waits.

Reply:
Done